target/linux/rockchip/image/rk3399.mk

```diff
@@ -0,0 +1,28 @@
+# 
+# Copyright (C) 2020 Tobias Maedel
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+define Device/pine64_rockpro64-emmc
+	DEVICE_VENDOR := Pine64
+	DEVICE_MODEL := RockPro64
+	DEVICE_VARIANT := emmc
+	DEVICE_DTS := rockchip/rk3399-rockpro64
+	SUPPORTED_DEVICES := pine64,rockpro64
+	UBOOT_DEVICE_NAME := rockpro64-rk3399
+	IMAGE/sysupgrade.img.gz := boot-common | boot-script | pine64-img | gzip | append-metadata
+endef
+
+define Device/pine64_rockpro64-sd
+	DEVICE_VENDOR := Pine64
+	DEVICE_MODEL := RockPro64
+	DEVICE_VARIANT := sd
+	DEVICE_DTS := rockchip/rk3399-rockpro64
+	SUPPORTED_DEVICES := pine64,rockpro64
+	UBOOT_DEVICE_NAME := rockpro64-rk3399
+	IMAGE/sysupgrade.img.gz := boot-common | boot-script | pine64-img | gzip | append-metadata
+endef
+
+TARGET_DEVICES += pine64_rockpro64-emmc
```

Comment:
Those should follow their definitions directly, with no newline between and one newline after. Just have a look at other targets if you are unsure.

Reply:
Fixed, thanks!